src/confd/src/ietf-interfaces.c

```diff
@@ -891,46 +885,47 @@ static int netdag_gen_veth(struct dagger *net, struct lyd_node *dif,
 static int netdag_gen_vlan(struct dagger *net, struct lyd_node *dif,
 			   struct lyd_node *cif, FILE *ip)
 {
-	const char *parent = lydx_get_cattr(cif, "parent-interface");
+	const char *lower_if;
 	const char *ifname = lydx_get_cattr(cif, "name");
 	struct lydx_diff typed, vidd;
-	struct lyd_node *otag;
+	struct lyd_node *vlan;
 	const char *proto;
 	int err;
 
-	DEBUG("ifname %s parent %s", ifname, parent);
-
-	err = dagger_add_dep(net, ifname, parent);
-	if (err)
-		return ERR_IFACE(cif, err, "Unable to add dep \"%s\"", parent);
-
-	otag = lydx_get_descendant(lyd_child(dif ? : cif),
-				   "encapsulation",
-				   "dot1q-vlan",
-				   "outer-tag",
+	vlan = lydx_get_descendant(lyd_child(dif ? : cif),
+				   "vlan",
 				   NULL);
```

Comment:
I think we can collapse this to a single line now, no need for multi-line.  (We also allow lines longer > 100 chars :)